e2e/tests/notebook/notebook-create-r.spec.ts

```diff
@@ -73,7 +73,7 @@ describe('Create R kernel notebook', () => {
       newNotebookName,
       ResourceCard.Notebook
     );
-    expect(modalTextContents).toContain(`Enter new name for ${cloneNotebookName}`);
+    expect(modalTextContents).toContain(`Enter new name for ${cloneNotebookName}.ipynb`);
```

Comment:
I think it would make more sense to have `cloneNotebookName` contain the `.ipynb`

Reply:
I think that is the only place need ipynb. For other `cloneNotebookName` in this file, seems it does not have suffix (`analysisPage.renameResourceFromTable` does not require suffix typed by user)